src/config/web3/cosmos/mainnet/chain4energy.ts

```diff
@@ -0,0 +1,43 @@
+import { Bech32Address } from "@keplr-wallet/cosmos";
+
+import { CosmosChain } from "../interface";
+
+export const chain4energy: CosmosChain = {
+  rpc: `https://rpc.c4e.io`,
+  rest: "https://lcd.c4e.io",
+  chainId: "perun-1",
+  chainName: "Chain4Energy",
```

Comment:
It is a bit confusing because the team asked us to onboard the chain in testnet with that name :
`/ $ axelard q nexus chains | grep chain
chains:
- chain4energy`

Afterwards they contacted us to onboard it in mainnet with a different name 
`/ $ axelard q nexus chains | grep c4
- c4e
/ $ `

The name in mainnet is C4E, here is what they asked us to onboard in mainnet : 
"Hello, I have an additional question, we are changing the name of our chain from chain4energy to c4e, do you use the chain registry? There will be no problem if we start with the name c4e?"